nodes/persistence-node/src/__tests__/jest/integration/ipfs-gateway-api.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { buildMainDependencyContainer } from "../../../modules/daemon/daemon.deps";
+import { Express } from "express";
+import supertest from "supertest";
+
+jest.setTimeout(50000);
+
+describe("ipfs-gateway-api", () => {
+  let app: Express;
+
+  const nonExistentHash = "QmcnrHegojMFqHkRhixazY67Zb9mSbMLv6sSxyDpUtnrQSy";
+  const nonBase32Hash = "SomeInvalidHash";
+  const genericErrorResponseMessage = "Something went wrong. Check the logs for more info.";
+
+  beforeAll(async () => {
+    const container = await buildMainDependencyContainer();
+    app = container.cradle.ipfsGatewayApi.createExpressApp();
+  });
+
+  it("cat returns 500 with message for unknown hash", async () => {
```

Comment:
@nerfZael Do we want to keep these tests now that we don't rely on error messages?